draft-wussler-openpgp-pqc.md

```diff
@@ -1214,6 +1233,12 @@ Algorithm ID reference | Parameter ID reference | Hash function | Hash function
 An implementation supporting a specific SPHINCS+ algorithm and parameter MUST
 also support the matching hash algorithm.
 
+### Key generation
+
+To generate a SPHINCS+ key the function `spx_keygen()` specified in
```

Comment:
```suggestion
The SPHINCS+ key generation is performed according to the function `spx_keygen()` specified in
```

Reply:
There was something missing like "the function is used" in the previous sentence.